FaceDetectionServer/moedx/facial_detection/tcp_server.py

```diff
@@ -100,6 +96,21 @@ def handle(self):
                         num_poses = len(poses2)
                         ret = {"success": "true", "server_processing_time": elapsed, "poses": poses2.tolist()}
 
+            elif opcode == 4: #'obj_det':
+                now = time.time()
+                image = imread(io.BytesIO(data))
+                pillow_image = Image.fromarray(image, 'RGB')
+                objects = myObjectDetector.process_image(pillow_image)
+                elapsed = "%.3f" %((time.time() - now)*1000)
```

Comment:
What does "%.3f" do?

Reply:
Formats a floating point number to 3 decimal places.